README.md

```diff
@@ -73,23 +73,83 @@ After you've made your working copy of the site repository, from its root folder
 hugo server -D
 ```
 
-## Test Links
+## Test Links and Alt Attributes
 
-You can test links with [htmltest](https://github.com/wjdp/htmltest). The `.htmltest.yml` includes options to
-avoid trailing slashes. The `htmltest` command works on the HTML content built
-in the public/ subdirectory.
+You can test links and alt text attributes with [htmltest](https://github.com/wjdp/htmltest). 
 
-After you fix broken links, run the following commands to rebuild content in
-the public/ subdirectory:
+### Install htmltest
+
+Follow the instructions to install `htmltest` in the /usr/local/bin folder. The current (1 Nov 2022)
+website suggests the following command, which works if you're using the `bash` shell:
+
+```
+sudo curl https://htmltest.wjdp.uk | sudo bash -s -- -b /usr/local/bin
+```
+
+If you're using the zsh shell (default in MacOS):
+
+```
+sudo curl https://htmltest.wjdp.uk | sudo zsh -s -- -b /usr/local/bin
```

Comment:
@ana-dashuk-cobalt I think I found the problem. Users need to make sure that they're in `sudo` (admin) mode when installing `htmltest`. You should be able to:


cd /path/to/cobalt-product-public-docs
sudo curl https://htmltest.wjdp.uk | sudo zsh -s -- -b /usr/local/bin
(Here's a screenshot of the messages that should appear -- )

<img width="923" alt="Screen Shot 2022-11-01 at 11 02 15 AM" src="https://user-images.githubusercontent.com/84352037/199305471-68e40464-a3d6-4f72-a7a5-2814af303e19.png">

hugo mod clean
hugo
htmltest

Reply:
@mjang-cobalt This instruction works for me, thank you!